README.md

```diff
@@ -2,30 +2,48 @@
 
 This repository houses the infrastructure for generating PDC SDKs.
 
-We're using a project called [swagger-codegen](https://github.com/swagger-api/swagger-codegen) to power our SDK, and this repository provides tooling and templates
-to make sdk generation easier in particular target languages. The codegen project supports a [many languages](https://github.com/swagger-api/swagger-codegen/tree/master/modules/swagger-codegen/src/main/resources)
-beyond the ones explicitly defined here.
+We're using a project called [swagger-codegen](https://github.com/swagger-api/swagger-codegen) to power our SDK, and this repository provides tooling and templates to make sdk generation easier in specifically supported languages.
 
 ## Prequisites
 
 - JRE (18+)
 
-## Generating sdks
-To generate the SDKs supported by this project, you will need:
+## Generating SDKs
 
-1. A [swagger](https://swagger.io/) specification located at the project root and named `openapi.json`. The current swagger specification for the PDC can be found in the [service repository](https://github.com/PhilanthropyDataCommons/service/blob/main/src/openapi.json).
+To generate the SDKs supported by this project, you will need a [swagger](https://swagger.io/) specification located at the project root and named `openapi.json`. The current swagger specification for the PDC can be found in the [service repository](https://github.com/PhilanthropyDataCommons/service/blob/main/src/openapi.json).
 
-2. Populated configuration files, for each language, located in the `configuration` directory.
+### Generating official SDKs:
 
-#### Steps:
+1. Clone the repository:
 
-From the project root you can run the following commands:
+```shell
+git clone git@github.com:PhilanthropyDataCommons/sdk.git
+```
+
+2. Change directory:
+
+```shell
+cd sdk
+```
+
+3. Copy your swagger specification
+
+```shell
+cp YOUR_SWAGGER_SPEC_LOCATION ./openapi.json
+```
+
+4. Generate the code bases
 
 ```shell
-> cp YOUR_SWAGGER_SPEC_LOCATION ./openapi.json
 > ./gradlew generateSwaggerCode
 ```
 
 The generated code will now exist in the `build` directory.
-Each language target will have its own directory inside of `build` (e.g. `build/typescript`).
-Some languages might have additional build steps defined in their build's README.md file.
+
+Each language target will have its own directory inside of `build`, and you can follow the readme instructions in that directory with additional steps related to that code base.
```

Comment:
```suggestion
Each language target will have its own directory inside of `build/`, and you can follow the readme instructions in that directory with additional steps related to that codebase.
```

Including the trailing slash is a convention I've adopted (and like!) when discussing directories.